source/Backend/DataBase/QueryOperationBuilder.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <format>
+#include <functional>
+#include <map>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+
+#include "common/DataBaseCommon.hpp"
+#include "common/DataBaseRequest.hpp"
+#include "interface/IQueryOperationBuilder.hpp"
+
+namespace dataBase {
+
+class QueryOperationBuilder : public interface::IQueryOperationBuilder {
+ public:
+  using QueryBuilders = std::unordered_map<
+      common::DataBaseRequest::OperationType,
+      std::function<std::string(const common::DataBaseRequest &,
+                                const std::string &)>>;
+  using OperationType = common::DataBaseRequest::OperationType;
+
+  QueryOperationBuilder() = default;
+  ~QueryOperationBuilder() = default;
+  std::string buildQuery(const common::DataBaseRequest &requestData) override;
+
+ private:
+  std::string buildInsertQuery(const common::DataBaseRequest &requestData,
+                               const std::string &tableName);
+  std::string buildUpdateQuery(const common::DataBaseRequest &requestData,
+                               const std::string &tableName);
+  std::string buildDeleteQuery(const common::DataBaseRequest &requestData,
+                               const std::string &tableName);
+  std::string buildSelectQuery(const common::DataBaseRequest &requestData,
+                               const std::string &tableName);
```

Comment:
Remove this whole section as it's no longer needed.

Reply:
thats right, I didn't knew if that was even possible :)